devel/cctools/Portfile

```diff
@@ -20,19 +24,23 @@ long_description        A set of essential tools to support development \
                         to binutils on other platforms.
 
 homepage                https://opensource.apple.com/source/${name}/
-master_sites            https://opensource.apple.com/tarballs/${name}:main \
-                        https://opensource.apple.com/tarballs/ld64:ld64
 
-distfiles               ld64-${ld64_version}.tar.gz:ld64 ${name}-${version}.tar.gz:main
+github.setup            apple-oss-distributions ${name} ${version}
```

Comment:
I'm wonder why have you used github PG if you're adding homepage and master_sites by userself?

Reply:
... at least for the downloads .. the homepage is to override the GH PG default

Reply:
@michaelld yeah, and I think that if you remove that PG, you won't lose anything :)